mordred/task_collection.py

```diff
@@ -81,9 +81,8 @@ def execute(self):
         clean = False
 
         fetch_cache = False
-        if ('fetch-cache' in cfg[self.backend_section] and
-            cfg[self.backend_section]['fetch-cache']):
-            fetch_cache = True
+        if ('no-cache' in cfg[self.backend_section]):
+            fetch_cache = not cfg[self.backend_section]['no-cache']
```

Comment:
Hmmm, @valeriocos `fetch_cache` configure gelk to get the data from the cache, and `no-cache` configure the backend to not store the data in cache. They are different options. Am I missing something?